### DISCUSSION TITLE
Låse drag and drop funksjonalitet til eier av tavle

### DISCUSSION THREAD

**Discussion by wfeiring** [entur/tavla#317] [Ideas]
Per i dag lagres posisjonen til kortene kun i localhost, som gjør at alle som får tilgang til en tavle-link har mulighet til å resize og endre rekkefølgen på kortene. Hadde vært fint å få knytta disse dataene til eieren av en tavla slik at det ikke kan endres dersom tavla det gjelder er låst.

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
